src/main/java/com/academy/terai/Controller/AccountResource.java

```diff
@@ -0,0 +1,54 @@
+package com.academy.terai.Controller;
+
+import com.academy.terai.Model.Account;
+import com.academy.terai.Repository.AccountRepository;
+import com.academy.terai.service.AccountService;
+import javassist.NotFoundException;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import javax.management.openmbean.KeyAlreadyExistsException;
+import java.util.List;
+
+@RestController
+@RequestMapping("/accounts")
+public class AccountResource {
+    private final AccountService accountService;
+    @Autowired
+    public AccountResource(AccountService accountService){
+        this.accountService = accountService;
+    }
+    @GetMapping(value = "/all")
+    public List<Account> getAllAccounts() {
```

Comment:
I'd suggest to wrap the response into ResponseEntity class - ResponseEntity<List<Account>>.
It's because ResponseEntity is a container for all REST response related data, including HTTP status, Headers and so on. Yes, you might won't add any now, but in REST area play with REST toys is better. Same applies to getAccountByEmail().